Tasks/cURLUploader/task.json

```diff
@@ -9,7 +9,7 @@
     "version": {
         "Major": 2,
         "Minor": 121,
```

Comment:
we should probably change the Minor version to this sprint number and leave the patch version at 0.